File: Fibonacci.go
package main

import (
  "fmt"
)

func Fibonacci(n int) {

  var n3, n1, n2 int = 0, 0, 1

  for i := 1; i <= n; i++ {

    fmt.Println(n1)

    n3 = n1 + n2

    n1 = n2

    n2 = n3

  }
}

func main() {

    Fibonacci(10)

  }

File: Scala/Fibonacci.scala
object FibonacciSeries {
  def main(args: Array[String]): Unit = {
    val limit: Int = 20 // Print the Fibonacci series

    var num1: Int = 0
    var num2: Int = 1

    println(s"Fibonacci Series upto $limit:")
    print(s"$num1 $num2")

    var sum: Int = num1 + num2
    while (sum <= limit) {
      print(" " + sum)

      num1 = num2
      num2 = sum
      sum = num1 + num2
    }
  }
}
